### ISSUE TITLE
Getting link error - undefined symbol llvm_pt_annotation_p0i8

### ISSUE THREAD

**Issue by Ujjwal21** [emscripten-core/emsdk#1106]
Hi team,

I am using emsdk version fastcomp-tag-1.38.30-64bit on my Linux system and using ninja build system. During linking, I am getting **undefined symbol error for "llvm_pt_annotation_p0i**8".

Can anyone please guide me how to debug this issue?

**Comment by sbc100** [comment]
The fastcomp compiler has been long deprecated.  Is there some reason you cannot the newer compiler (`emsdk install latest`)?

**Comment by Ujjwal21** [comment]
@sbc100 yes we tried moving onto latest emsdk but there were issues in moving to latest, I will get back here with exact issues. But we have been using this fastcomp compiler since some time now, any idea why we could start getting this issue?

**Comment by sbc100** [comment]
No idea, that fastcomp compiler hasn't been updated in years.

**Comment by Ujjwal21** [comment]
Any idea which function/pointer trigger this intrinsic. Eg "memcpy" in C, during link time is @llvm.memcpy.p0i8

**Comment by sbc100** [comment]
You would need to search the llvm codebase for llvm_ptr_annotation.. but remember that factcomp is based on an ancient fork of llvm.

I would suggest that you spend you efforts trying to update to latest emsdk.

**Comment by sbc100** [comment]
Its been a long time now since we stopped supporting fastcomp, so closing this old issue.
